released/discovery_center/mission_3774/step2/sample.tfvars

```
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------------------------------------------
+# Provider configuration
+# ------------------------------------------------------------------------------------------------------
+# Your global account subdomain
+globalaccount        = "yourglobalaccount"
+region               = "datacenter"
+subaccount_name      = "subaccount_name"
+cf_environment_label = "cf_environment_label"
+cf_api_url           = "cf api url"
+
+# ------------------------------------------------------------------------------------------------------
+# Project specific configuration (please adapt!)
+# ------------------------------------------------------------------------------------------------------
+
+cfsr_space_manager        = "john.doe@test.com"
```

The `cfsr_` prefix is not correct. This should be `cf_` in accordance to the variables and it must be a list:

```suggestion
cf_space_manager        = [ "john.doe@test.com" ]
```